### PR TITLE
[MIG] product_weight: Migration to 13.0

### PR DESCRIPTION
**Pull request by tonygalmiche**
Do not review at this time. I'm just starting the migration.

### CODE REVIEW

File: product_weight/tests/test_compute_bom_weight.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2015 Akretion (<http://www.akretion.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestBomWeightCompute(TransactionCase):
+    def setUp(self):
+        super(TestBomWeightCompute, self).setUp()
```
Comment (bealdav) [inline product_weight/tests/test_compute_bom_weight.py]: In python 3

```suggestion
        super().setUp()
```

File: product_weight/wizard/product_weight_update.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    product_tmpl_id = fields.Many2one("product.template", "Template")
+    product_id = fields.Many2one("product.product", "Product")
+    bom_id = fields.Many2one(
+        "mrp.bom", "BoM", domain="[('product_tmpl_id', '=', product_tmpl_id)]"
+    )
+
+    @api.model
+    def default_get(self, fields):
+        res = super(ProductWeightUpdate, self).default_get(fields)
```
Comment (bealdav) [inline product_weight/wizard/product_weight_update.py]:
```suggestion
        res = super().default_get(fields)
```

File: product_weight/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# Copyright (C) 2015 Akretion (<http://www.akretion.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (bealdav) [inline product_weight/models/__init__.py]: Headers in an empty file are useless

File: product_weight/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+* Joao Alfredo Gama Batista <ermitaoj@gmail.com>
+* Florian da Costa <florian.dacosta@akretion.com>
+* Juan Humanes <juan.humanes@guadaltech.es>
+* Thore Baden <thorebaden@gmail.com>
```
Comment (bealdav) [inline product_weight/readme/CONTRIBUTORS.rst:4]: @tonygalmiche you should add you name/mail here

**Review by bealdav — CHANGES_REQUESTED** [review]
Thanks for your contribution

**Comment by tonygalmiche** [comment]
Thank you @bealdav for your review. It's corrected

**Review by bealdav — APPROVED** [review]

**Review by hparfr — APPROVED** [review]
LGTM

**Comment by tonygalmiche** [comment]
Thank you @hparfr  for your review

**Comment by pedrobaeza** [comment]
@dreispt ocabot stuck on merge

**Comment by rousseldenis** [comment]
> @dreispt ocabot stuck on merge

Relaunched it. Switching to travis github app should solve webhook trigger problems like this.

**Pull request merged**